### ISSUE TITLE
Tab focus should be constrained within Hot graphic button popup.

### ISSUE THREAD

**Issue by robertmausser** [adaptlearning/adapt-contrib-hotgraphic#228]
If keyboard focus moves out of the popup, users would not realize that the content is not a part of the pop-up.

https://www.screencast.com/t/5RFK3FfgvnW

Solution: "Construct a modal window/ popup which should be accessible. Focus should be trapped inside modal when it is open.
 For details refer: https://www.w3.org/TR/wai-aria-practices/examples/dialog-modal/dialog.html"

**Comment by oliverfoster** [comment]
https://www.w3.org/TR/wai-aria-practices/examples/dialog-modal/dialog.html

This is exactly what you're witnessing. The site content beneath the dialog is not accessible, that is the focus trap.

We can not and should not stop the user from tabbing to the browser navigation.
